package.json

```diff
@@ -32,6 +32,7 @@
     "bounding-client-rect": "1.0.5",
     "browser-filesaver": "1.1.0",
     "chalk": "1.0.0",
+    "check-node-version": "2.0.1",
```

Comment:
This might be a typo: the [latest version](https://www.npmjs.com/package/check-node-version) is `2.1.0`.